ts/webui/src/components/managementExp/Experiment.tsx

```diff
@@ -0,0 +1,433 @@
+import * as React from 'react';
+import { Stack, DetailsList, DefaultButton, Icon, SearchBox, IColumn } from '@fluentui/react';
+import { ExperimentsManager } from '../../static/model/experimentsManager';
+import { expformatTimestamp, copyAndSort } from '../../static/function';
+import { AllExperimentList, SortInfo } from '../../static/interface';
+import MessageInfo from '../modals/MessageInfo';
+import { compareDate, filterByStatusOrPlatform, getSortedSource } from './expFunction';
+import { MAXSCREENCOLUMNWIDHT, MINSCREENCOLUMNWIDHT } from './experimentConst';
+import { Hearder } from './Header';
+import NameColumn from './NameColumn';
+import FilterBtns from './FilterBtns';
+import '../../App.scss';
+import '../../static/style/nav/nav.scss';
+import '../../static/style/experiment/experiment.scss';
+import '../../static/style/tableStatus.css';
+
+interface ExpListState {
+    columns: IColumn[];
+    platform: string[];
+    errorMessage: string;
+    hideFilter: boolean;
+    searchInputVal: string;
+    selectedStatus: string;
+    selectedPlatform: string;
+    selectedStartDate?: Date;
+    selectedEndDate?: Date;
+    sortInfo: SortInfo;
+    source: AllExperimentList[];
+    originExperimentList: AllExperimentList[];
+    searchSource: AllExperimentList[];
+}
+
+class Experiment extends React.Component<{}, ExpListState> {
```

Comment:
Suggest to rename to `ExperimentManager` or something similar. Current name looks like "one experiment"..